etcdutl/etcdutl/backup_command.go

```diff
@@ -322,7 +322,7 @@ func saveDB(lg *zap.Logger, destDB, srcDB string, idx uint64, term uint64, desir
 
 	if !v3 {
 		tx := be.BatchTx()
-		tx.Lock()
+		tx.LockInsideApply()
```

Comment:
I have already changed it to `LockOutsideApply`.  Actually it doesn't matter which method it calls, because the txPostLockHook is nil, see [batch_tx.go#L80](https://github.com/ahrtr/etcd/blob/8259df98235f9887fe022e583bd56e71d88a5e6c/server/storage/backend/batch_tx.go#L80)